src/Signer/OpenSSL.php

```diff
@@ -56,15 +59,20 @@ final protected function createSignature(
     }
 
     /** @throws CannotSignPayload */
-    private function getPrivateKey(string $pem, string $passphrase): OpenSSLAsymmetricKey
-    {
+    private function getPrivateKey(
+        #[SensitiveParameter]
+        string $pem,
+        #[SensitiveParameter]
+        string $passphrase,
+    ): OpenSSLAsymmetricKey {
         return $this->validateKey(openssl_pkey_get_private($pem, $passphrase));
     }
 
     /** @throws InvalidKeyProvided */
     final protected function verifySignature(
         string $expected,
         string $payload,
+        #[SensitiveParameter]
```

Comment:
I apologize for my previous statement: verification operations only deal with public keys, and by definition public keys don't neet to be protected.

We can rever changes to both `verifySignature` and `getPublicKey`